src/test/resources/FindUser.feature

```diff
@@ -0,0 +1,31 @@
+Feature:
+  As a user
+  I want to be find another user
+
+  Background:
+    Given database is empty
+    And user with id 1234 exists and has username "kangaroo", password "pig-system" and refreshToken "token1"
+    And user with id 1235 exists and has username "penguin", password "i-love-capitalism" and refreshToken "token2"
+    
+  Scenario: Successful find another user
+    When user with id "1234" and token "token1" searches user with search-value "penguin"
+    Then response status code is 200
+    And response contains key "username" and value "penguin"
+    And response contains key "id" and value 1235
```

Comment:
You can use the step `response contains the user with id {long}` if you want to check if the user is returned.